config/initializers/decidim.rb

```diff
@@ -91,7 +91,7 @@
   #   end
   # end
   #
-  # config.sms_gateway_service = 'Decidim::Verifications::Sms::ExampleGateway'
+  config.sms_gateway_service = "Decidim::SmsGatewayService"
```

Comment:
```suggestion
if Rails.application.secret.dig(:decidim, :sms_gateway_service).present?
  config.sms_gateway_service = Rails.application.secret.dig(:decidim, :sms_gateway_service)
end
```

I expect to be able to define the sms_gateway_service using env variable like `ENV.fetch("SMS_GATEWAY_SERVICE", nil)`

Reply:
Hello Quention, thx fo your suggestion, I made the changes 😃